### DISCUSSION TITLE
[Question] Any way to skip posting of the message?

### DISCUSSION THREAD

**Discussion by arhont375** [danger/danger-js#1119] [Q&A]
Hello,
I want to disable posting of PR comment if, Danger have zero messages to deliver to user. Is there a way to do this?

**Comment by orta** [comment]
This is the default behavior, so maybe you have some CLI flags set to enable this?

**Reply by arhont375** [reply]
Thanks for quick answer!
Apparently my problem was that I was setting `markdown` message (the one that appending at the end) and expecting that it would be shown only when there is some other problems, but seems presence of this message also will trigger PR comment.
